kolibri/plugins/learn/assets/src/views/ContentPage.vue

```diff
@@ -334,6 +337,25 @@
       updateContentState(contentState, forceSave = true) {
         this.updateContentNodeState({ contentState, forceSave });
       },
+      navigateToRegularContext(message) {
+        let id = message.nodeId;
+        return ContentNodeResource.fetchModel({ id })
+          .then(contentNode => {
+            let routeBase, path;
+            if (contentNode && contentNode.kind !== 'topic') {
+              routeBase = '/topics/c';
+              path = `${routeBase}/${id}`;
+              router.push({ path: path }).catch(() => {});
+            } else if (contentNode && contentNode.kind === 'topic') {
+              routeBase = '/topics/t';
+              path = `${routeBase}/${id}`;
+              router.push({ path: path }).catch(() => {});
+            }
+          })
+          .catch(err => {
+            console.log(err);
```

Comment:
Should use the standard API error handling logic here if there's an unexpected error.